pkg/bgpv1/types/bgp.go

```diff
@@ -69,7 +69,8 @@ type GetBGPResponse struct {
 
 // ServerParameters contains information for underlying bgp implementation layer to initializing BGP process.
 type ServerParameters struct {
-	Global BGPGlobal
+	Global     BGPGlobal
+	OnFIBEvent func()
```

Comment:
nit: FIB is slightly misleading term. Route learned from BGP is not really gone into FIB ( linux routing table ). I'd suggest `OnRouteEvent`.